test/test_project.py

```diff
@@ -405,6 +405,17 @@ def test_import_exception(self):
         exc = assert_context.exception
         self.assertIn("error", exc.args[0])
 
+    @responses.activate
+    def test_import_metadata(self):
```

Comment:
I'm happy with this test 👍🏻

Reply:
I have been experimenting with additional tests and found some weird behavior, of which I suspect is is due to the test setup as it works with other redcap servers. Do you have an idea why the following test might fail?
```python
    @responses.activate
    def test_import_reduced_metadata(self):
        "Test import of a reduced set of metadata"
        self.add_normalproject_response()
        data = self.reg_proj.export_metadata()
        # reducing the metadata
        data = data[0:1]
        d = self.reg_proj.import_metadata(data)
        # this should return the reduced set of metadata, but instead returns the full set of metadata when running on the test server
        self.assertEqual(len(d), len(data))
```